src/pages/CourtInfo.jsx

```diff
@@ -16,37 +16,96 @@ function CourtInfo() {
       </div>
 
       <Form>
-        <Form.Group className="mb-3" controlId="question1">
-          <Form.Label>Queston 1</Form.Label>
+        <Form.Group className="mb-3" controlId="Name:" required>
+          <Form.Label>Question 1<span className="text-danger fs-4">*</span></Form.Label>
           <Form.Control type="text" placeholder="Short answer" />
           <Form.Text className="text-muted">
             Add additional instructions if needed here.
           </Form.Text>
         </Form.Group>
```

Comment:
For each question I think you've mixed up the `<Form.Label>` and the `controlID`
The `<Form.Label>` holds the 'title' of the question, we probably want that to read Name: in this case rather than Question 1
`controlID` should be unique for each question posed, these could be question1, question2... as in the original (no spaces)

This is true for each question

Reply:
I have fixed this issue